src/main/studio/ui/EditorPane.java

```diff
@@ -129,4 +129,9 @@ private void setBorder(JComponent component) {
                 )
         );
     }
+
+    public void setFont2(Font font) {   //don't call this setFont, it leads to an error
```

Comment:
Technically such method is not required as it is possible to execute pane.getTextArea().setFont().
Or if you think we need to encapsulate the logic here, I would suggest to name the method `setTextAreaFont` (tbh, setFont2 looks confusing to me)